### DISCUSSION TITLE
May I ask for the architecture information about Strix Point Mini PC: Beelink SER9 : AMD Ryzen™ AI 9 HX 370

### DISCUSSION THREAD

**Discussion by ngdymx** [Xilinx/mlir-aie#1910] [Q&A]
Dear team,

I recently purchased this new computer but can’t find its datasheet. Could you provide me with the details of its architecture? Thank you very much!

**Comment by jgmelber** [comment]
Hi! Please send me an email, I would like to chat about your projects and goals. joseph.melber@amd.com
